### DISCUSSION TITLE
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'generation_expression' in 'field list'

### DISCUSSION THREAD

**Discussion by bhattji** [filamentphp/filament#13657] [Help]
### Package

Form builder

### Package Version

V3.2.95

### How can we help you?

I ran the command
php artisan make:filament-resource Location --generate --soft-deletes

it gives error

Illuminate\Database\QueryException 

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'generation_expression' in 'field list' (Connection: mysql, SQL: select column_name as `name`, data_type as `type_name`, column_type as `type`, collation_name as `collation`, is_nullable as `nullable`, column_default as `default`, column_comment as `comment`, generation_expression as `expression`, extra as `extra` from information_schema.columns where table_schema = 'bhattji_far_bca' and table_name = 'locations' order by ordinal_position asc)

    809▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    810▕                 );
    811▕             }
    812▕
  ➜ 813▕             throw new QueryException(
    814▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    815▕             );
    817▕     }

  1   vendor\laravel\framework\src\Illuminate\Database\Connection.php:407
      PDOException::("SQLSTATE[42S22]: Column not found: 1054 Unknown column 'generation_expression' in 'field list'")

  2   vendor\laravel\framework\src\Illuminate\Database\Connection.php:407
      PDO::prepare()

Please help

**Comment by leandrocfe** [comment]
`php artisan about --only=Filament`
What is the output?

**Reply by bhattji** [reply]
Filament .................................................................................  
  Packages ................................. filament, forms, notifications, support, tables  
  Version .......................................................................... v3.2.96  
  Views ...................................................................... NOT PUBLISHED

**Comment by xujiongze** [comment]
Same problem, with output
  Filament .........................................................................................................................................  
  Packages ......................................................................................... filament, forms, notifications, support, tables  
  Version .................................................................................................................................. v3.2.95  
  Views .............................................................................................................................. NOT PUBLISHED

**Comment by avalon1412** [comment]
that problem may because database you are using. generation_expression is avaiable on mysql version 5.7 or newer
